packages/backend/src/scheduler/SchedulerTask.ts

```diff
@@ -1721,18 +1772,44 @@ export default class SchedulerTask {
                                 gdriveId,
                                 chartNames[chartUuid] || chartUuid,
                             );
+                        const pivotConfig = getPivotConfig(chart);
```

Comment:
This is to support `pivot tables` when uploading ghseets on dashboard scheduler. 

We currently don't support `sync on dashboards`, but I updated the code to support this  when we enable the feature.